app/src/main/res/layout/item_skill_group_list.xml

```diff
@@ -2,23 +2,22 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/group_parent"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="50dp"
```

Comment:
Dont hard code dimens value

Reply:
should i create new string... I have to approximate it if i want to use existing

Reply:
Create a new string